src/test/java/uk/ac/bath/se/DungeonTest.java

```diff
@@ -1,17 +1,25 @@
 package uk.ac.bath.se;
 
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.BeforeClass;
 import org.junit.Test;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
 public class DungeonTest {
+    @Test
+    public void testInitialiseDungeon() throws Exception {
+        Dungeon dungeon = Dungeon.getInstance();
+        dungeon.initialiseDungeon();
+        String[][] dungeonMatrix = dungeon.getDungeonMatrix();
+        assertTrue(dungeonMatrix != null);
+        assertTrue(dungeon.getHeight() == dungeonMatrix.length);
```

Comment:
Should this be dungeonMatrix[1] to follow the line below?

Reply:
no first is how long the array is the second is how many arrays each array contains